examples/capi-quick-start/aws-cluster-calico-crs.yaml

```diff
@@ -53,3 +53,15 @@ spec:
         - class: default-worker
           name: md-0
           replicas: ${WORKER_MACHINE_COUNT}
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta2
+kind: AWSClusterStaticIdentity
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: aws
+  name: ${CLUSTER_NAME}
+spec:
+  allowedNamespaces:
+    list:
+      - default
+  secretRef: aws-quick-start-creds
```

Comment:
I don't think these are being used - https://github.com/d2iq-labs/capi-runtime-extensions/blob/main/charts/capi-runtime-extensions/defaultclusterclasses/aws-cluster-class.yaml#L34-L38

Reply:
Ahh yep you're right, removed all of that